pkg/gptscript/gptscript.go

```diff
@@ -80,6 +84,9 @@ func Complete(opts ...Options) Options {
 	if len(result.CredentialContexts) == 0 {
 		result.CredentialContexts = []string{credentials.DefaultCredentialContext}
 	}
+	if result.DatasetToolRepo == "" {
+		result.DatasetToolRepo = defaultDatasetToolRepo
+	}
```

Comment:
I don't think this is needed given that you have a default set.

I do believe it is needed here, though, when running the SDK server in embedded mode: https://github.com/gptscript-ai/gptscript/blob/45d444f810600d9584951ec3aba98da9b5e02db5/pkg/sdkserver/server.go#L154

Reply:
Thanks. This is fixed